server/routes/induction/common/hopingToWorkOnReleaseController.ts

```diff
@@ -18,6 +18,11 @@ export default abstract class HopingToWorkOnReleaseController extends InductionC
   ): Promise<void> => {
     const { prisonerSummary, inductionDto } = req.session
 
+    if (req.session.pageFlowHistory) {
```

Comment:
Because the "Hoping to work on release" page is the first question in the Induction there was never a "previous" page, so the concept of page history for this page was not a thing.
This now changes because we have the circumstance where we DO have a history, in that the previous page might be the Check Your Answers page, hence this change